### PR TITLE
name/email/password editing

### PR DESCRIPTION
**Pull request by Leminode**
Changes:
- changes to the user profile page
- new UserProfileService
- temporarily added users table to dbContext, not yet sure if we'll use it later

Known issues:
- Redirect after successful password change does not work
- UserName and UserEmailAddress field validation does not work

### CODE REVIEW

**Review by Tauras-Rim — APPROVED** [review]
good job

File: Energy Saver/Pages/Account/Profile.cshtml.cs [hunk 1]
```diff
@@ -42,31 +50,57 @@ public void OnGet()
...
-            RestResponse response = client.Execute(request);
-            _logger.LogInformation(response.Content);
+            //OLD IMPLEMENTATION (DO NOT REMOVE YET):
 
-            OnProfileEditSuccess();
+            //var client = new RestClient("https://priolette.eu.auth0.com/api/v2/");
+            //var request = new RestRequest($"/users/{userId}", RestSharp.Method.Patch);
 
-            return RedirectToPage("./Profile");
+            //request.AddHeader("content-type", "application/json");
```
Comment (priolette) [inline Energy Saver/Pages/Account/Profile.cshtml.cs:71]: Will you still work with these commented lines? If not, then they should probably be deleted

**Review by priolette — APPROVED** [review]

**Review by nocheacosador — APPROVED** [review]
Nice

File: Energy Saver/Pages/Account/Profile.cshtml.cs (hunk 1 above)
Comment (Leminode) [reply Energy Saver/Pages/Account/Profile.cshtml.cs:71]: i might still need them in the future for a few things, so i'll keep them right now

**Pull request merged**